pages/index.md

```diff
@@ -4,4 +4,43 @@ layout: home
 nav_order: 1
 ---
 
-The Redwire Labs guide for developing commercial products with [Nerves](https://nerves-project.org/).
+This is a guide to developing commercial products with [Nerves](https://nerves-project.org/)
+based on the lessons learned by the team at [Redwire Labs](https://www.redwirelabs.com/).
+Product development can be a challenging journey, so we are sharing this
+resource with the community so that you can avoid the pitfalls we have seen
+and shortcut the path to success.
+
+*Redwire Labs is a product development agency that specializes in commercial
+IoT products built with Nerves and Elixir.*
+
+## Why we use Nerves
+
+In our experience, Nerves has been an excellent technology that expedites the
+development of connected products, and is built on a platform that supports
+robustness and reliability.
+
+Nerves is the framework for running an [Elixir](https://elixir-lang.org/)
+application on embedded Linux. It uses [Buildroot](http://buildroot.org/) to
+create a custom Linux system that's tailored to your project and boots the
+system into the application (as process 1), opposed to running a Linux distro,
+which tends to be heavier weight and comes with extra cruft. Nerves also
+provides facilities to update firmware in the field, and an A/B partition scheme
```

Comment:
Optional: maybe this reads better?

```suggestion
provides facilities to update firmware in the field with an A/B partition scheme
```

Reply:
This was intentionally two different items. Updates are with `fwup` and are a separate feature from A/B partitions.